tests/unit/amo/sagas/test_reviews.js

```diff
@@ -577,5 +602,85 @@ describe(__filename, () => {
 
       expect(action).toEqual(expectedAction);
     });
+
+    it('flashes the start of saving a rating', async () => {
```

Comment:
is "to flash" a verb? I always used "flash" as an adjective for "flash messages" (as opposed to persistent messages).

Reply:
I always thought of it as a verb in this context to say *flash a message on the screen temporarily*.